boards/gardena/sgrm/doc/index.rst

```diff
@@ -0,0 +1,113 @@
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+
+Connections and IOs
+===================
+
++--------+--------------------------+----------------------------------------------------+
+| Pin    | Name                     | Note                                               |
++========+==========================+====================================================+
+| PB0.0  | TX (O)                   | Serial connection to the Linux SoM                 |
++--------+--------------------------+                                                    |
+| PB0.1  | RX (I)                   |                                                    |
++--------+--------------------------+                                                    |
+| PB0.2  | RTS (O)                  |                                                    |
++--------+--------------------------+                                                    |
+| PB0.3  | CTS (I)                  |                                                    |
++--------+--------------------------+----------------------------------------------------+
+| PB0.4  | LED red (I)              | Controlled by the Linux SoM                        |
++--------+--------------------------+                                                    |
+| PB0.5  | LED green (I)            |                                                    |
++--------+--------------------------+                                                    |
+| PB0.6  | LED blue (I)             |                                                    |
++--------+--------------------------+----------------------------------------------------+
+| PB0.13 | TX (O)                   | UART1 for debugging (no connection to Linux SoM)   |
++--------+--------------------------+                                                    |
+| PB0.14 | RX (I)                   |                                                    |
++--------+--------------------------+----------------------------------------------------+
+| PB4.0  | LED red (O)              | Mirrors PB0.4                                      |
++--------+--------------------------+----------------------------------------------------+
+| PB4.1  | LED green (O)            | Mirrors PB0.5                                      |
++--------+--------------------------+----------------------------------------------------+
+| PB4.2  | LED blue (O)             | Mirrors PB0.6                                      |
++--------+--------------------------+----------------------------------------------------+
+
+Programming and Debugging
+*************************
+
+Flashing
+========
+
+The easiest way is to do this via SSH from the Linux SoM that's connected to the SiM3U SoM.
+
+On your building machine:
+
+.. code-block:: console
```

Comment:
```suggestion
.. code-block:: shell
```
https://docs.zephyrproject.org/latest/contribute/documentation/guidelines.html#code-blocks-and-command-examples
Please check other blocks too

Reply:
fixed